stb_vorbis.c

```diff
@@ -553,7 +553,7 @@ enum STBVorbisError
 #include <string.h>
 #include <assert.h>
 #include <math.h>
-#if !(defined(__APPLE__) || defined(MACOSX) || defined(macintosh) || defined(Macintosh))
```

Comment:
Most platforms (including Linux) have `malloc()`, `free()` and related in `<stdlib.h>`, conforming to C99, so maybe a positive check for the platforms actually needing malloc.h instead of stdlib.h would be better?